samples/apps/logging/js/src/logging.js

```diff
@@ -256,6 +256,47 @@ export function post_private(request) {
   return { body: true };
 }
 
+export function post_private_admin_only(request) {
+  // Return an auth error if the caller has no user data,
+  // or the user data is not an object with isAdmin field,
+  // or this field is not true
+  const data = request.caller.data;
+  if (!(typeof data === "object" && data !== null && data.isAdmin === true)) {
```

Comment:
I believe you can do this:

```suggestion
  if (!(data?.isAdmin === true)) {
```